lib/public/Model.js

```diff
@@ -179,6 +181,9 @@ export default class Model extends Observable {
             case 'run-overview':
                 this.runs.loadOverview();
                 break;
+            case 'runs-per-period':
```

Comment:
Why does the name of the URL do not match the model & other stuff?

Reply:
changed